src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/FeedConstants.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.DotNet.DarcLib.Helpers
+{
+    /// <summary>
+    /// Constants related to package feed management
+    /// </summary>
+    public class FeedConstants
+    {
+        public static readonly string MaestroManagedFeedNamePattern = @"darc-(?<type>(int|pub))-(?<repository>.+?)-(?<sha>[A-Fa-f0-9]{7,40})-?(?<subversion>\d*)";
+
+        public static readonly string[] MaestroManagedFeedPatterns =
```

Comment:
Worth using `MaestroManagedFeedNamePattern` interpolated into the strings below (don't know if the feed name parts are the same)?

Reply:
Yeah I was actually thinking about that and I'm getting that to work now. That way we only have the name pattern in one place if we ever decide to change it.